### DISCUSSION TITLE
Manage account settings in mobile app

### DISCUSSION THREAD

**Discussion by trustytrojan** [community/community#48813] [Mobile]
Reduce the need for me to go to github.com and implement an account settings menu

**Comment by mattruggio** [comment]
Hi @trustytrojan, thanks for the feedback!  Could you list some settings you find yourself having to access github.com for?  This could potentially help us identify some of the more important settings in future planning.

**Reply by trustytrojan** [reply]
One thing is editing my public profile. This is an action I cannot perform on Github Android.

**Reply by mattruggio** [reply]
@trustytrojan - That makes a lot of sense to me.  I am going to put this into an internal issue for the mobile team to further discuss.

**Comment by van4elo86** [comment]
> Reduce the need for me to go to github.com and implement an account settings menu

@davidstaheli
